content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,227 @@
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`Sendcloud Configuration` tab, enter the :guilabel:`Sendcloud Public Key`.
+- In the :guilabel:`Sendcloud Configuration` tab, enter the :guilabel:`Sendcloud Secret Key`.
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`Sendcloud Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Load your Sendcloud shipping products` link.
+- Select the shipping products the company would like to use for deliveries and returns.
+- Click :guilabel:`Select`.
+
+.. example::
+   This is an example of the Sendcloud shipping products configured in Odoo.
+
+   | **Delivery:**
+   | Shipping Product: DPD Home 0-31.5kg
+   | Carrier: DPD
+   | Minimum Weight: 0.00
+   | Maximum Weight: 31.50
+
+   Countries: Austria, Belgium, Bosnia and Herzegovina, Bulgaria, Croatia, Czech Republic,
+   Denmark, Estonia, Finland, France, Germany, Greece, Hungary, Iceland, Ireland, Italy, Latvia,
+   Liechtenstein, Lithuania, Luxembourg, Monaco, Netherlands, Norway, Poland, Portugal, Romania,
+   Serbia, Slovakia, Slovenia, Spain, Sweden, Switzerland
+
+   | **Return:**
+   | Return Shipping Product: DPD Return 0-20kg
+   | Return Carrier: DPD
+   | Return Minimum Weight: 0.00
+   | Return Minimum Weight: 20.00
+   | Return Countries: Belgium, Netherlands
+
+.. image:: sendcloud_shipping/sendcloud-example.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
+
+.. tip::
+   Sendcloud does not provide test keys when a company tests the sending of a package in Odoo. This
+   means if a package is created, the configured Sendcloud account will be charged, unless the
+   associated package is canceled within 24 hours of creation.
+
+   Odoo has built an extra layer of protection into test environments. If the shipping method is
+   used to create a label, then the labels are immediately canceled after the creation. This occurs
+   automatically. The test and production environments can toggled back and forth from the
+   :guilabel:`Smart Buttons`.
+
+Generate a label with Sendcloud
+-------------------------------
+
+When creating a quotation in Odoo, add shipping and a :guilabel:`Sendcloud shipping product`. Then,
+:guilabel:`Validate` the delivery. Shipping label documents are automatically generated in the
+chatter.
+
+The following should be included in the shipping label documents:
+
+#. :guilabel:`Shipping label(s)` (depending on the number of packages).
+#. :guilabel:`Return label(s)` (if the Sendcloud connector is configured for returns).
+#. :guilabel:`Customs document(s)` (should the destination country require them).
+
+Additionally, the tracking number is also available.
```

Comment:
`Additionally` and `also` in the same sentence is redundant.

Assuming that the tracking number is generated _after_ the sales order is validated:

```suggestion
Additionally, the tracking number is now available.
```